.github/workflows/test_on_push_bricks-and-hearts.yml

```diff
@@ -12,6 +12,21 @@ jobs:
   build:
     name: Build and run tests
     runs-on: ubuntu-latest
+    
+    services:
+      sqlserver:
+        image: mcr.microsoft.com/mssql/server:2019-latest
+        env:
+          ACCEPT_EULA: Y
+          SA_PASSWORD: JdMsKZPBBA8kVFXVrj8d
+        ports:
+          - 1433:1433
+        options: >-
+          --health-cmd "/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P 'JdMsKZPBBA8kVFXVrj8d' -Q 'SELECT 1' || exit 1"
+          --health-interval 10s
+          --health-timeout 3s
```

Comment:
I'd probably lower the interval a bit, and up the retries, no need to wait 10s

Reply:
I've adjusted the parameters, hopefully this is more reasonable